app/src/main/java/org/sagebionetworks/research/mpower/profile/ProfileFragment.java

```diff
@@ -73,36 +79,9 @@ public void onDestroyView() {
     @OnClick(R.id.licenses_button)
     public void onLicensesClicked() {
         LOGGER.debug("Clicked on the licenses button");
-        String path =  MpResourceManager.getInstance().getLicense().getAbsolutePath();
+        String path = MpResourceManager.getInstance().getLicense().getAbsolutePath();
         Intent intent = org.researchstack.backbone.ui.ViewWebDocumentActivity.newIntentForPath(this.getActivity(),

```

Comment:
We probably should have used a WebView instead of adding a dependency on ResearchStack's ResourceManager and ViewWebDocumentActivity

Reply:
I think the issue was more figuring out *how* to get the resource to load into the web view.